hotdoc/extensions/gst/gst_extension.py

```diff
@@ -864,7 +864,7 @@ def __create_property_symbols(self, obj, parent_uniquename,
             type_ = QualifiedSymbol(type_tokens=tokens)
 
             default = prop.get('default')
-            if obj['hierarchy'][0] != parent_uniquename:
```

Comment:
I'm missing some context here, can you elaborate on why this is needed / give an example in the commit message?